src/app/components/Curation/index.test.tsx

```diff
@@ -39,15 +39,4 @@ describe('Topic Curations', () => {
       expect(getByTestId(testId)).toBeInTheDocument();
     },
   );
-
-  it('should render the standard grid if a style/prominence is not recognised', () => {
-    const { getByTestId } = render(
-      <Curation
-        visualStyle="something-unsupported"
-        visualProminance="what-is-this"
```

Comment:
Deleted because Typescript wouldn't allow me to use style and prominence that were not in the list